frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -160,12 +163,45 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
     return <LoadingBox />;
   }
 
+  const handleChartVersionChange = (chart: HelmChart) => {
+    setChartReadme(getHelmChartReadme(chart));
+  };
+
+  const launchReadmeModal = () => {
+    import('./HelmReadmeModal')
+      .then((m) =>
+        m.helmReadmeModalLauncher({
+          readme: chartReadme,
+          modalClassName: 'modal-lg',
+        }),
+      )
+      .catch((e) => {
+        throw e;
+      });
+  };
+
   return (
     <NamespacedPage variant={NamespacedPageVariants.light} disabled hideApplications>
       <Helmet>
         <title>{config.title}</title>
       </Helmet>
-      <PageHeading title={config.title}>{chartHasValues && config.subTitle}</PageHeading>
+      <PageHeading title={config.title}>
+        {chartHasValues && config.subTitle}
+        {chartReadme && (
+          <>
+            &nbsp; For more information on the chart, refer to this
```

Comment:
`&nbsp; ` will cause extra whitespace at the start of the line if `chartHasValues == false` as the line above won't display a `subTitle`.

Reply:
fixed